python/src/aiconfig/editor/client/src/components/prompt/PromptContainer.tsx

```diff
@@ -12,8 +12,8 @@ import ModelSelector from "./ModelSelector";
 
 type Props = {
   prompt: ClientPrompt;
-  cancel: (cancellationToken: string) => Promise<void>;
-  getModels: (search: string) => Promise<string[]>;
+  cancel?: (cancellationToken: string) => Promise<void>;
```

Comment:
nit: for future diff, can we call this `onCancel` so it's more obvious this is a callback? Not a blocker

Reply:
Oh nm, I see that this is just the method itself, not a callback that invokes other functionality. Interesting